packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -199,8 +182,8 @@ class _TreemapState extends State<Treemap> {
         );
 
         // Calculate the aspect ratio for the pivot treemap node.
-        final pAndList2Ratio = (pSize + list2Size) / totalSize;
-        final pRatio = pSize / (pSize + list2Size);
+        final pAndList2Ratio = (pByteSize + list2Size) / totalByteSize;
```

Comment:
avoid abbreviations in names. For example: use 
`pivot` instead of `p` in variable names. That makes it easier for someone new to the code to tell what the variables mean. Here I had to look up a line to browse up to the previous lines to figure out what `p` was.